lib/response_bank/middleware.rb

```diff
@@ -36,10 +36,10 @@ def call(env)
             body.each { |part| body_string << part }
           end
 
-          body_gz = ResponseBank.compress(body_string)
+          body_br = ResponseBank.compress(body_string, 'br')
 
           # Store result
-          cache_data = [status, headers['Content-Type'], body_gz, timestamp]
```

Comment:
what about when cloudflare only sends `Accept-Encoding: gzip`?

Reply:
With the latest commit, response bank supports these scenarios:

* CF sends `Accept-Encoding: gzip`, if server cache miss, render the page, compress and store in cache; if server cache hit (gzip encoded payload), return compressed payload.
* CF sends `Accept-Encoding: br`, if server cache miss, render the page, compress and store in cache; if server cache hit (br encoded payload), return compressed payload.
* CF sends a request without `Accept-Encoding` header or a different encoding `Accept-Encoding: deflate`, I don't know if this is possible in reality, but response bank will also support it. In this case `env['encoding']` is `nil`, `env['server_cache_encoding']` is `br`. Response bank will retrieve `br` key, if cache hit, decompress the payload and return raw page; if cache miss, render the page, and compress in `br`, and return raw page.